packages/react/src/Avatar/Avatar.figma.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import Avatar from './Avatar'
+import figma from '@figma/code-connect'
+
+/**
+ * -- This file was auto-generated by `figma connect create` --
+ * `props` includes a mapping from Figma properties and variants to
+ * suggested values. You should update this to match the props of your
+ * code component, and update the `example` function to return the
+ * code example you'd like to see in Figma
+ */
+
+figma.connect(
+  Avatar,
+  'https://www.figma.com/file/GCvY3Qv8czRgZgvl1dG6lp/Primer-Web?type=design&node-id=3805-11&mode=design&t=n5UtEmPJTCzDEJKK-4',
```

Comment:
Just curious, are these URLs found by going to the Primer Web figma file and finding the specific component or is there something else to it? 👀

Reply:
Yes, this is exactly how to do it.